composables/useUiHelpers/index.ts

```diff
@@ -0,0 +1,19 @@
+type Category = {
+  categoryId: Number
+  categoryCode: String
+}
+
+interface uiHelpersReturnType {
+  getCatLink?: (category: Category) => string
+}
+
+const useUiHelpers = (): uiHelpersReturnType => {
+  const getCatLink = (category: Category) => {
```

Comment:
We should reuse the "Category" type from the graphql Category

Reply:
resolved. use PrCategory from graphql